music21/harmony.py

```diff
@@ -2499,6 +2500,57 @@ def transpose(self: NCT, _value, *, inPlace=False) -> NCT | None:
 
 # ------------------------------------------------------------------------------
 
+class FiguredBass(Harmony):
+    '''
+    The FiguredBassIndication objects store information about thorough bass figures.
+    It is created as a representation for <fb> tags in MEI and <figured-bass> tags in MusicXML.
+    The FiguredBassIndication object derives from the Harmony object and can be used
+    in the following way:
+    >>> fb = harmony.FiguredBass('#,6#', correspondingPart='P1')
+    >>> fb
+    <FiguredBass figures: #,6# correspondingPart: P1>
+
+    The single figures are stored as figuredBass.notation.Figure objects:
+    >>> fb.figNotation.figures[0]
+    <music21.figuredBass.notation.Figure 3 <Modifier # sharp>>
+    '''
+
+
+    def __init__(self,
+                 figureString: str | list[str] | None = None,
+                 correspondingPart: str | None=None,
+                 **keywords):
+        super().__init__(**keywords)
+
+        self.isFigure: bool = True
+        self.corresPart: str | None = None
+        self._figs: str = ''
+
+        if figureString:
+            if isinstance(figureString, list):
+                self._figs = ','.join(figureString)
+            elif isinstance(figureString, str):
+                if ',' in figureString:
+                    self._figs = figureString
+                else:
+                    self._figs = ','.join(figureString)
+        else:
+            self._figs = ''
+        self._figNotation = notation.Notation(self._figs)
+        self.corresPart = correspondingPart
+
+    @property
+    def figNotation(self) -> notation.Notation:
+        return self._figNotation
+
+    @figNotation.setter
+    def figNotation(self, figs: str | list[str] | None):
+        self._figNotation = notation.Notation(figs)
```

Comment:
as noted above, this doesn't pass reflexive setters.  (also no abbreviations.  This should just be `notation`)

Needs docs and tests.

it seems like there should be two sets of properties.  One for getting/setting the Notation object, and one for getting/setting the string of figures -- changing one should change the other.

Reply:
Good idea. I'm pushing a proposal.